### ISSUE TITLE
Plugin doesn't work if ideal is not selected

### ISSUE THREAD

**Issue by stenvdb** [craftcms/commerce-mollie#18]
Hi,

In the `getPaymentFormHtml()` method there is a check if the `fetchIssuers()` method works. But this method does not work if ideal is not selected in the website profile in mollie. Therefore, if ideal is not selected the payment form html is not returned.

**Comment by stenvdb** [comment]
Maybe this could be useful for others:

I didn't use the `getPaymentFormHtml()` in the end. What I did instead was:

```
{% for paymentMethod in cart.gateway.fetchPaymentMethods() %}
  {# Implement a custom radio button here #}
  {% if paymentMethod.id == 'ideal' %}
    {% for issuer in cart.gateway.fetchPaymentMethods() %}
      {# Implement a custom dropdown here #}
    {% endfor %
  {% endif %}
{% endfor %}

**Comment by andris-sevcenko** [comment]
> In the getPaymentFormHtml() method there is a check if the fetchIssuers() method works. But this method does not work if ideal is not selected in the website profile in mollie.

Okay, but what would be the resolution here? If Mollie doesn't return the issuers then we don't have it. Seems like your approach is a great way as it's also more likely that the site developer will know what the list has to be.

**Comment by stenvdb** [comment]
Hi @andris-sevcenko, issuers and payment methods are 2 different things in mollie. If issuers don't apply (-> when ideal is not checked in mollie's dashboard), then we should still be returning the selected payment methods. But in the current implementation nothing is returned.

**Comment by andris-sevcenko** [comment]
Gotcha!

**Comment by timvermaercke** [comment]
This is because `fetchIssuers()` throws a `Omnipay\Common\Http\Exception\RequestException`, as there are no issuers defined for Belgian payment methods. The `getPaymentFormHtml`method catches the exception and returns an empty response.

I created a pull request with my fix;

**Comment by nfourtythree** [comment]
This is fixed with the merge of #30 .

Fix will be included in the next release of the plugin.

Thanks!
